src/main/java/uk/gov/hmcts/reform/idam/client/models/TokenExchangeResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.idam.client.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TokenExchangeResponse {
+
+    @JsonProperty("access_token")
```

Comment:
are you sure Jackson will use this? (as there will be a public getter)

Reply:
Yes it will use it
https://github.com/hmcts/cmc-claim-store/blob/master/src/main/java/uk/gov/hmcts/cmc/claimstore/idam/models/TokenExchangeResponse.java